### ISSUE TITLE
Reports of causing Red Trustfactor

### ISSUE THREAD

**Issue by AutismWarrior** [danielkrupinski/VAC-Bypass#72]
There are multiple people on Unknowncheats saying VAC Bypass gives players Red trustfactor on CSGO, is this true?

My account is in red trustfactor now, but it could be that I have been reported multiple times due to going blatant because of cheaters.

**Comment by dexyycs** [comment]
Yes, I had a red trust factor and I was not playing matchmaking at ALL. Now I have a VAC ban, not an overwatch ban that was transformed into a VAC. I would not cheat rn as they are literally doing something against it.
